scripts/upgrade.d/upgrade.11.sh

```diff
@@ -0,0 +1,13 @@
+gitlab_version="11.11.5"
+
+gitlab_x86_64_source_sha256="1ee3d6e8d2cc198f5466de0884c03f6016299db24859126af9a191501dbdef10"
+
+gitlab_arm_source_sha256="366e12b1f3d3b1694fcb6f13da9de908360ba93f75768d97e8d01e61e8652705"
+
+gitlab_filename="gitlab-ce-${gitlab_version}.deb"
+
+# Action to do in case of failure of the package_check
+package_check_action() {
+        ynh_replace_string --match_string="command \"cat \/etc\/sysctl.conf \/etc\/sysctl.d\/\*.conf  | sysctl -e -p -\"" --replace_string="command \"cat \/etc\/sysctl.conf\"" --target_file="$final_path/embedded/cookbooks/package/resources/sysctl.rb"
+}
+
```

Comment:
Is it really useful to declare this function here if you have to declare it again in each upgrade script ?
It seems easier to just use that line in the install script and the upgrade script.

Reply:
It's not the same action for the version 11.x and 12.x:
```bash
 		ynh_replace_string --match_string="command \"sysctl -e --system\"" --replace_string="command \"sysctl -e --system || true\"" --target_file="$final_path/embedded/cookbooks/package/resources/sysctl.rb"
```

So during the upgrade I use the good one for the upgrade I do.

Reply:
Indeed, haven't read the code after `ynh_replace_string`.